api/document/js/path-to-resolved-pos.ts

```diff
@@ -9,6 +9,9 @@ export class NthType {
   constructor(nth: number, nodeType: string | NodeType) {
     this.nth = nth;
     this.nodeType = nodeType instanceof NodeType ? nodeType : schema.nodes[nodeType];
+    if (!this.nodeType) {
+      throw new Error(`"${nodeType}" is not a valid node in the schema`);
```

Comment:
This isn't a necessary change for this PR, but I figured it'd be more useful feedback than something more cryptic down the line, when something else tried accessing an attribute of `nodeType`.